public/css/common.less

```
@@ -95,23 +95,26 @@ div.show-more {
     margin-bottom: .5em;
   }
 
+  #host-meta-info {
+    margin-bottom: 0;
+  }
+
+  .host-meta-info {
+    margin-bottom: .5em;
+  }
```

Please put it inside, so that the bottom margin is only applied if it's inside the `#host-meta-info` div.

```suggestion
  #host-meta-info {
    margin-bottom: 0;

    .host-meta-info {
      margin-bottom: .5em;
    }
  }
```

---

✅